.appveyor.yml

```diff
@@ -1,6 +1,7 @@
 skip_branch_with_pr: true
 
 environment:
+  ANDROID_HOME: "C:\\android-sdk"
```

Comment:
Do we really need the double-backslash (and double quotes) here? At least the YAML also [validates](https://ci.appveyor.com/tools/validate-yaml) without.

Reply:
Both examples I found used the double backslash, if you want I can try without it once the rest works.

Reply:
I'm curious, let's try if it works without `\\` and `"`.

Reply:
It does.